verification/VerifyTasks.wdl

```diff
@@ -133,4 +133,43 @@ task CompareCrais {
     memory: "2 GiB"
     preemptible: 3
   }
+}
+
+task CompareBams {
+
+  input {
+    File test_bam
+    File truth_bam
+  }
+
+  Float bam_size = size(test_bam, "GiB") + size(truth_bam, "GiB")
+  Int disk_size = ceil(bam_size * 4) + 20
+
+  command {
+    set -e
+    set -o pipefail
+
+    java -Xms3500m -jar /usr/picard/picard.jar \
+    CompareSAMs \
+          ~{test_bam} \
+          ~{truth_bam} \
+          O=comparison.tsv \
+          LENIENT_HEADER=true
+
+    if [ $? -eq 0 ]
```

Comment:
Why is this block of code needed?  The exit code of the invocation of CompareSAMs should be the exit code of the command block since it's the last command issued.

Reply:
Thanks! Took this code block out and reran the tests expecting them to fail, and they did and the code exited just fine 👍 

https://gotc-jenkins.dsp-techops.broadinstitute.org/job/warp-workflow-tests/4372/console